### PR TITLE
Deprecate use of term slave

### PR DESCRIPTION
**Pull request by bdmorin**
Please consider removing the term slave.
Some alternatives might be:
 - minion
 - operative
 - node
 - worker
 - replica
 - drone
 - handler
 - steward
 - emissary
 - doer
 - mason
 - coordinator
 - conscript

Also consider adoption of a contributor covenant:
https://www.contributor-covenant.org/

Thanks.

### CODE REVIEW

File: doc/install/agents.rst [hunk 1]
```diff
@@ -18,10 +18,10 @@ On the "master", install IVRE following the
 ``screen``, ``tmux`` or ``nohup`` if you want to be able to "detach"
 from the ``agent`` script (which is not a daemon).
 
-On the "slave(s)", the ``agent`` script must be deployed, together with
+On the "remotehost(s)", the ``agent`` script must be deployed, together with
```
Comment (p-l-) [inline doc/install/agents.rst]:
```suggestion
On the "worker(s)", the ``agent`` script must be deployed, together with
```

File: doc/install/agents.rst [hunk 2]
```diff
@@ -18,10 +18,10 @@ On the "master", install IVRE following the
...
 ``nmap``, and ``rsync``.
 
-Run the slave(s)
+Run the remotehost(s)
```
Comment (p-l-) [inline doc/install/agents.rst]:
```suggestion
Run the worker(s)
```

**Comment by p-l-** [comment]
Since the project includes its own documentation (rendered), the change has to be propagated. I have included this to #907.

**Comment by bdmorin** [comment]
Thank you!!!

**Pull request closed without merging**